libcudacxx/include/cuda/__memory_resource/cuda_memory_resource.h

```diff
@@ -155,6 +155,9 @@ struct cuda_memory_resource
 };
 static_assert(resource_with<cuda_memory_resource, device_accessible>, "");
 
+//! @brief globally available instance of a ``cuda_memory_resource``
+_LIBCUDACXX_CPO_ACCESSIBILITY cuda_memory_resource default_memory_resource{};
```

Comment:
Suggestion: `default_memory_resource` is ambiguous.

```suggestion
_LIBCUDACXX_CPO_ACCESSIBILITY cuda_memory_resource default_cuda_memory_resource{};
```

Reply:
Arguably should also apply to the other two.

Reply:
Dropped the global instances as they indeed might give us a lot of pain later on